pypdf/_cmap.py

```diff
@@ -75,21 +75,12 @@ def build_char_map_from_dict(
         for x in int_entry:
             if x <= 255:
                 encoding[x] = chr(x)
-    # I consider the space_code is available on one byte
     if isinstance(space_code, str):
-        try:  # one byte
```

Comment:
Why can we leave this out?

Reply:
In parse_encoding this could be “ ” and the corresponding number.

In parse_to_unicode we get “ ” and the corresponding key character taken from map_dict. That is the value we want.

It compares the character with a numeric chr conversion or the key character from map_dict as an argument to the character with a chr conversion in Widths, etc.

I think the erased code is unnecessary. Also, I still think it would be easier to understand if the whitespace conversion and processing were separated.

Reply:
If it is difficult to determine, we will revert the code for the part of the code that is being used as the character code.
I will refactor parse_encoding and parse_to_unicode at a later date, so you can decide at that time.